packages/std/src/query.rs

```diff
@@ -24,7 +25,6 @@ pub enum QueryRequest<C: CustomQuery> {
         /// this is the expected protobuf message type (not any), binary encoded
         data: Binary,
     },
-    Wasm(WasmQuery),
```

Comment:
What's the idea behind this move? Before it was alphabetically. I'm happy either way but I'd like to understand how you want to order this.

Reply:
I left it alphabetic for the standard ones, then all the stargate ones below (under feature flag). So sorting (is_stargate, name).

Maybe that is not very good idea, as staking is also under a different feature flag. I can make it alphabetical again

Reply:
I'd either use plain alphabetic order or introduce ordered groups that are separated by an empty line